docs/src/content/docs/api/tools.mdx

```diff
@@ -0,0 +1,628 @@
+---
+title: "Tools"
+sidebar:
+  order: 30
+---
+import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
+
+Tools in Ultravox allow you to extend the capabilities of your AI assistant by giving it access to external APIs and services. With tools, your AI can perform actions like fetching real-time data, interacting with databases, or integrating with third-party services, all while maintaining a natural conversation flow.
+
+### Two Types of Tools
+Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call.
+
+Iteration is faster when using temporary tools. Durable tools shine once you have things dialed in, when you want to share tools across multiple applications, or if you have split responsibilities on the team.
+
+### Creating Tools
+Temporary tools are defined and passed in the request body of the [`Create Call`](../calls/#create-call) endpoint.
+
+You create durable tools either by uploading an OpenAPI spec or via the request body in the [`Create Tool`](./#create-tool) endpoint. Your OpenAPI spec must be either `json` or `yaml` format.
+
+## List Tools
+<Badge text="GET /tools" variant="note" size="large" />
+
+Lists all available tools. API key scoped.
```

Comment:
I thought we had decided on "Account scoped"?

Reply:
yes you are correct. i must not have cleaned all those up before and so they got pulled in. will fix.